src/screens/MainApp/Home/Dashboard/DiagnosesBoard/BoardItem/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { View, Text, Image } from 'react-native'
+import styles from './styles'
+import cannabisLeaf from './resources/placeholder.png'
+import answeredIcon from './resources/answered.png'
+import notAnsweredIcon from './resources/notAnswered.png'
+
+
+const RealThumbnailOrPlaceholder = ({ thumbnail }) => (
+  <Image
+    source={thumbnail ? { uri: thumbnail } : cannabisLeaf}
+    style={styles.thumbnail}
+  />
+)
+
+const AnsweredIcon = ({ answered }) => {
+  const icon = (answered === false) ? notAnsweredIcon : answeredIcon
+  const style = (answered === false) ? styles.notAnswered : styles.answered
+  return (<Image source={icon} style={style} />)
+}
```

Comment:
- No need to test if it equals `false`. The following is more readable:
```javascript
  const icon = answered ? answeredIcon : notAnsweredIcon
  const style = answered ? styles.answered : styles.notAnswered
```
- No need to use parenthesis on one-line return statements.